pkg/encoding/v1/encoding.go

```diff
@@ -319,6 +320,9 @@ func (d *Decoder) Decode(data []byte) (*object.OpenCompose, error) {
 			Name: string(s.Name),
 		}
 
+		// user given value of replicas
+		os.Replicas = s.Replicas
```

Comment:
using values as it is here and checking down whether it should land in `deployment` or not? thoughts: @tnozicka